cross/cups/Makefile

```diff
@@ -22,7 +22,7 @@ CONFIGURE_ARGS = --with-cupsd-file-perm=0755 \
         --with-xinetd=no \
         --with-access-log-level=actions \
         --enable-page-logging \
-	'LDFLAGS=-Wl,-rpath,$(STAGING_INSTALL_PREFIX)/lib -Wl,--rpath,$(INSTALL_DIR)/var/packages/$(PKG_NAME)/target/lib'
+	LDFLAGS="$(LDFLAGS) -Wl,-rpath,$(STAGING_INSTALL_PREFIX)/lib -Wl,-rpath,$(INSTALL_DIR)/var/packages/$(PKG_NAME)/target/lib"
```

Comment:
you can use `ADDITIONAL_LDFLAGS` - and LDFLAGS already contains `-Wl,-rpath,$(STAGING_INSTALL_PREFIX)/lib`

`ADDITIONAL_LDFLAGS = -Wl,-rpath,$(INSTALL_DIR)/var/packages/$(PKG_NAME)/target/lib`

_and please use `CONFIGURE_ARGS += ...`  to concatenate multiple lines_